oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/logging/LoggingConfigWatcher.java

```diff
@@ -63,14 +64,6 @@ public void notify(final ConfigChangeEvent value) {
             log.error("failed to apply configuration to log4j", t);
             return;
         }
-        StringBuilder builder = new StringBuilder();
-        ctx.getConfiguration().getLoggers().forEach((loggerName, config) -> {
-            builder.append(Strings.isNullOrEmpty(loggerName) ? "Root" : loggerName)
-                   .append(":")
-                   .append(config.getLevel())
-                   .append(",");
-        });
-        this.content = builder.toString();
```

Comment:
@hanahmily This was a bug of old codes, which cause endless loop of update notification.